README.md

```diff
@@ -1 +1,136 @@
-# OOP-school-library
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo (Not Microverse logo but your own name logo) at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍